legate/core/launcher.py

```diff
@@ -407,26 +533,30 @@ def coalesce(self, error_on_interference):
         else:
             return [pair for pair in self._entries.items()]
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return str(self._entries)
 
 
 class FieldSet:
-    def __init__(self):
-        self._fields = {}
+    def __init__(self) -> None:
+        self._fields: dict[int, ProjectionSet] = {}
 
-    def insert(self, field_id, perm, proj_info):
+    def insert(
+        self, field_id: int, perm: Permission, proj_info: EntryType
+    ) -> None:
         if field_id in self._fields:
             proj_set = self._fields[field_id]
         else:
             proj_set = ProjectionSet()
             self._fields[field_id] = proj_set
         proj_set.insert(perm, proj_info)
 
-    def coalesce(self, error_on_interference):
-        coalesced = {}
+    def coalesce(
+        self, error_on_interference: bool
+    ) -> dict[Any, list[Union[int, FieldID]]]:
+        coalesced: dict[Any, list[Union[int, FieldID]]] = {}
         for field_id, proj_set in self._fields.items():
-            proj_infos = proj_set.coalesce(error_on_interference)
+            proj_infos = proj_set.coalesce(error_on_interference)  # line 496
```

Comment:
I assume you're gonna remove the comment, right?

Reply:
bizarre, I have no idea where this came from. https://github.com/nv-legate/legate.core/pull/189/commits/7bc51b3a159364102d2a8a3fbd2049bbbe585d5c